server/schemas/typeDefs.js

```diff
@@ -2,7 +2,7 @@ const { gql } = require('apollo-server-express');
 
 const typeDefs = gql`
     type Movie {
-        movieId: String
+        movieId: Int
```

Comment:
Is this intentional?

If not, could we leave it this way? I realized that js will convert strings into integers to do the comparison, and since the response gives us an integer we should continue to use int for movieId